cscs-checks/prgenv/affinity_check.py

```diff
@@ -17,6 +18,7 @@ def __init__(self, variant):
         # The github URL can not be specifid as `self.sourcedir` as that
         # would prevent the src folder from being copied to stage which is
         # necessary since these tests need files from it.
+        self.sourcesdir += os.path.join('/affinity_ref')
```

Comment:
```suggestion
        self.sourcesdir = os.path.join(self.sourcesdir, 'affinity_ref')
```

And similarly in the rest.

Reply:
done